### ISSUE TITLE
Field generated from `array_agg()` producing error

### ISSUE THREAD

**Issue by PatilShreyas** [medly/norm#65]
Whenever `array_agg()` is used in SQL, it's producing field which is causing error while building project.

**For e.g.:**

We have below function in our SQL query:
```sql
array_agg(p.rx_number) rx_numbers
```

It's generating field as below:

![image](https://user-images.githubusercontent.com/19620536/89274725-ed9d4b80-d65e-11ea-935e-4417edd9a8fc.png)

As you can see in the above image, extra `>` is generated at the end which is the root cause of the error.

*Currently we need to solve it manually to build the project successfully as below:*
```diff
- rxNumbers = rs.getArray("rx_numbers").array as kotlin.Array<kotlin.String>?>,
+ rxNumbers = rs.getArray("rx_numbers").array as kotlin.Array<kotlin.String>?,
```

**Comment by bhimsenp** [comment]
@PatilShreyas, this issue has already been fixed in version 0.0.5

**Comment by PatilShreyas** [comment]
Okay thanks @bhimsenp
